Diffing_Engine/Query/ListModifiedProperties.cs

```diff
@@ -39,8 +39,8 @@ public static partial class Query
 
         [MultiOutput(0, "identifier", "Identifier of the objects which have some modified properties.\nWhen using Revisions, this is the Hash of the objects. When Diffing using CustomData, this is the specified Id.")]
         [MultiOutput(1, "propNames", "List of properties changed per each object.")]
-        [MultiOutput(2, "value_Current", "List of current values of the properties.")]
-        [MultiOutput(3, "value_Past", "List of past values of the properties.")]
+        [MultiOutput(2, "value_obj1", "List of current values of the properties.")]
+        [MultiOutput(3, "value_obj2", "List of past values of the properties.")]
```

Comment:
```suggestion
        [MultiOutput(2, "currentValues", "List of current values of the properties.")]
        [MultiOutput(3, "pastValues", "List of past values of the properties.")]
```

Reply:
This suggestion is more helpful to the user I think in terms of what the outputs are (based on their descriptions)